src/components/Button.js

```diff
@@ -1,9 +1,22 @@
 import styled from 'styled-components/macro';
 
 export const ButtonStyle = styled('button')`
+  transform: translate(0%);
+  transition: ${props => props.theme.transition};
   color: ${props => props.theme.fg};
-  border: 2px solid ${props => props.theme.fg};
+  border: ${props => (props.buttonBorder ? `1px solid ${props => props.theme.fg}` : 0)};
```

Comment:
Another way of doing this: 

```js
import styled, { css } from 'styled-components/macro'

export const ButtonStyle = styled('button')`
  (...)
  ${props => props.buttonBorder && css`
    border: 1px solid ${props => props.theme.fg};
   }
  (...)
```

In this case, both are perfectly fine ways of doing it **and this doesn't require to be changed if you don't want to**. But when you have `2` or more rules that require the same if-statement, that's when using the `css` templating really shines!